src/views/pages/globalping/network-tools.html

```diff
@@ -223,6 +223,36 @@ <h1>{{displayingTestType}} from multiple locations in </h1>
 	const PROBE_STATUS_OFFLINE = _.getProbeStatusOfflineValue();
 
 	component.exports = {
+		computed: {
+			titleAndDescr () {
```

Comment:
I don't understand why this was moved to be a computed property if it's still used just once during the load.